src/lib/plugin-loader.ts

```diff
@@ -67,60 +69,84 @@ export default function loadPlugin<T extends IPlugin<T>>(
         const externalFilePlugin = Path.resolve(pluginDir, pluginId);
         plugin = tryLoad(externalFilePlugin);
 
+        loadedId = pluginId;
+        from = 'external plugin folder';
+
         // npm package
         if (plugin === null && pluginId.match(/^[^\.\/]/)) {
           plugin = tryLoad(Path.resolve(pluginDir, `${prefix}-${pluginId}`));
+          loadedId = `${prefix}-${pluginId}`;
+
           // compatibility for old sinopia plugins
           if (!plugin) {
             plugin = tryLoad(Path.resolve(pluginDir, `sinopia-${pluginId}`));
+            loadedId = `sinopia-${pluginId}`;
+          }
+
+          // compatibility for new @verdaccio/ plugins
+          if (!plugin) {
+            plugin = tryLoad(Path.resolve(pluginDir, `@${prefix}/${pluginId}`));
+            loadedId = `@${prefix}/${pluginId}`;
           }
         }
       }
 
       // npm package
       if (plugin === null && pluginId.match(/^[^\.\/]/)) {
         plugin = tryLoad(`${prefix}-${pluginId}`);
+        loadedId = `${prefix}-${pluginId}`;
+        from = 'locally installed plugin';
+
         // compatibility for old sinopia plugins
         if (!plugin) {
           plugin = tryLoad(`sinopia-${pluginId}`);
+          loadedId = `sinopia-${pluginId}`;
+        }
+
+        if (!plugin) {
```

Comment:
I'm running some test, but I could not make this work with the example you shared in the ticket. In theory here you would get `@verdaccio/@verdaccio/active-directory`.

Reply:
If @verdaccio/active-directory is installed you could set it up either as `active-directory` or `@verdaccio/active-directory`.

First case: it would be loaded in line 108.
Second case: it would be loaded in line 113.

Reply:
I'm still looking into this. Just playing around. Most likely we would need add test. I hope can back with feedback soon.